components/automate-chef-io/data/docs/api-static/02-meta-description.yaml

```diff
@@ -35,14 +41,14 @@ info:
     export TOKEN2=bww8EEpr39_eYMnQ2zybtrP9uzk=
     ```
 
-    You can grant a standard token access to specific parts of the Chef Automate API 
-    if you write a policy. 
+    You can grant a standard token access to specific parts of the Chef Automate API
+    if you write a policy.
     The [Permission for An API Client](https://automate.chef.io/docs/authorization-overview/#permission-for-an-api-client)
     section of the Automate documentation walks through how to grant a token access to all Compliance API endpoints.
 
     For more information about authentication, refer to the
     [API Tokens](https://automate.chef.io/docs/api-tokens/)
-    section of the Automate documentation. 
-    For more information about authorization, refer to the 
-    [Authorization Overview](https://automate.chef.io/docs/authorization-overview/#overview) 
+    section of the Automate documentation.
+    For more information about authorization, refer to the
+    [Authorization Overview](https://automate.chef.io/docs/authorization-overview/#overview)
```

Comment:
oh this link just changed, its now
https://automate.chef.io/docs/iam-v1-overview/#overview

but there is a redirect in place